deployment/lib/README

```diff
@@ -0,0 +1,2 @@
+This folder came from "https://github.com/kbase/kbapi_common/tree/master/lib" 
```

Comment:
Is this file generated by getKBaseLibs.sh in Makefile.mini?

Reply:
Yeah, let me delete that from the repo